technical-reference.adoc

```diff
@@ -1,512 +1,466 @@
 | Connection Idle Timeout Unit a| Enumeration, one of:
- 
-** NANOSECONDS
-** MICROSECONDS
-** MILLISECONDS
-** SECONDS
-** MINUTES
-** HOURS
-** DAYS |  Time unit for the *Connection Idle Timeout* field. |  SECONDS |
+
+** `NANOSECONDS`
+** `MICROSECONDS`
+** `MILLISECONDS`
+** `SECONDS`
+** `MINUTES`
+** `HOURS`
+** `DAYS` |  Time unit for the *Connection Idle Timeout* field. |  `SECONDS` |
 | Stream Response a| Boolean |  If this value is `true`, Mule streams received responses. |  false |
 | Response Buffer Size a| Number |  Size of the buffer that stores the HTTP response, in bytes. |  -1 |
-| Reconnection a| <<Reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
+| Reconnection a| <<reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
 |===
- 
+
 == Operations
-* <<SubmissionsCreateJson>>
-* <<SubmissionsList>>
-* <<SubmissionsRetrieve>>
-* <<Unauthorize>>
- 
- 
-[[SubmissionsCreate]]
-== Submission Create Json
+* <<submissions-create-json>>
+* <<submissions-list>>
+* <<submissions-retrieve>>
+* <<unauthorize>>
+
+
+[[submissions-create-json]]
+== Submission Create JSON
 `<hyperscience:submissions-create-json>`
- 
- 
-Creates a submission using JSON payload. This operation makes an HTTP POST request to the api/v5/submissions endpoint.
- 
+
+
+Creates a submission using a JSON payload. This operation makes an HTTP POST request to the `api/v5/submissions` endpoint.
+
 === Parameters
- 
-[%header%autowidth.spread]
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | Name of the configuration to use. | | x
-| Body a| Any | Content to use. |  #[payload] |
-| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
-| Streaming Strategy a| * <<RepeatableInMemoryStream>>
-* <<RepeatableFileStoreStream>>
-* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
-| Custom Query Parameters a| Object | Custom query parameters to include in the request. The specified query parameters are merged with the default query parameters that are specified in the configuration. |  |
-| Custom Headers a| Object | Custom headers to include in the request. The specified custom headers are merged with the default headers that are specified in the configuration. |  |
+| Configuration | String | `Name` of the configuration to use. | | x
+| Body a| Object | Content to use. |  `#[payload]` |
+| Config Ref a| ConfigurationProvider |  `Name` of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* <<non-repeatable-stream>> |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
```

Comment:
@rui-lomba , what type of repeatable-stream is the default? We have 2.

Reply:
file-store, added!
Thanks